server/asset/dcr/dcr.go

```diff
@@ -963,7 +966,10 @@ func (dcr *Backend) getUnspentTxOut(ctx context.Context, txHash *chainhash.Hash,
 func (dcr *Backend) getTxOutInfo(ctx context.Context, txHash *chainhash.Hash, vout uint32) (*chainjson.GetTxOutResult, *chainjson.TxRawResult, []byte, error) {
 	verboseTx, err := dcr.node.GetRawTransactionVerbose(ctx, txHash)
 	if err != nil {
-		return nil, nil, nil, fmt.Errorf("GetRawTransactionVerbose for txid %s: %w", txHash, translateRPCCancelErr(err))
+		if isTxNotFoundErr(err) { // since we're calling gettxout after this, check now
+			return nil, nil, nil, asset.CoinNotFoundError
+		}
```

Comment:
I'm at a loss to understand how we did without this.  Today I was suddenly getting errors from server when placing orders with txsplit, such that the server is expected not to see the coin initially and keep searching for a short period.  This definitely used to work, but it's clear that `getTxOutInfo` requires this check because unlike on the btc backend, we are calling getrawtransaction *before* gettxout.  The order router was rejecting orders because the error `FundingCoin` was not `asset.CoinNotFound`.

I'd really like to know when this broke, but it was very recent because I quite often use txsplit in simnet and testnet.

Reply:
Tracked down the change to https://github.com/decred/dcrdex/commit/bc4e19932d4ba242b55035e10284483eab84aada#diff-70ea3126faa1f8216d381a32b3c2bc13238608b0045a5316d83949e091d2dff7L953-L961
The gettxout call was moved after the sendrawtransaction call.  I've confirmed that this is not an issue on `release-v0.2` so no need to backport this server/asset/dcr change, although I do intend to add the client/asset/btc/btc.go to `release-v0.2`.